src/varity/hgvs_to_vcf.clj

```diff
@@ -33,6 +33,10 @@
   [hgvs seq-rdr rgs]
   (distinct (apply concat (keep #(prot/->vcf-variants hgvs seq-rdr %) rgs))))
 
+(defn- ->supported-transcript
+  [s]
+  (re-find #"^((NM|NR)_|ENS(T|P))\d+(\.\d+)?$" s))
```

Comment:
I think this fn name and the return value are discord. Furthermore, this fn has only to check the transcript.

Recommendation:
```clj
(defn- supported-transcript?
  [s]
  (some? (re-matches #"((NM|NR)_|ENS(T|P))\d+(\.\d+)?" (str s))))
```